engine-rest/engine-rest/src/test/java-jersey2/org/camunda/bpm/engine/rest/util/container/JerseyServerBootstrap.java

```diff
@@ -61,7 +61,8 @@ private void setupServer(Application application) {
     int port = Integer.parseInt(serverProperties.getProperty(PORT_PROPERTY));
     URI serverUri = UriBuilder.fromPath(ROOT_RESOURCE_PATH).scheme("http").host("localhost").port(port).build();
     try {
-      server = GrizzlyHttpServerFactory.createHttpServer(serverUri, rc);
+      server = GrizzlyHttpServerFactory.createHttpServer(serverUri, rc, false);
+      server.getServerConfiguration().setAllowPayloadForUndefinedHttpMethods(true);
```

Comment:
Not sure if we should do this since it might be possible that some of our supported environments are configured differently by default. Let's add an integration test to see if the endpoint works on all environments if we want to do this anyway.

Reply:
I will probably go for a path parameter for the date then if that is more widely supported. Should save us some headaches., right?

Reply:
I agree. Like this, we don't risk that the endpoint works in some environments and others not.